src/components/Item/Item.tsx

```diff
@@ -210,7 +210,7 @@ function ItemMenuButton({
       ) : (
         <button
           onClick={
-            showMenu
+             showMenu as unknown as React.MouseEventHandler<HTMLButtonElement>
```

Comment:
Haha, fun times. Yeah, we can come back and clean stuff like this up after all the major refactoring is done